modules/AWS/rds/main.tf

```diff
@@ -40,20 +40,29 @@ module "db" {
   allocated_storage = var.allocated_storage
   iops              = var.iops
 
-  name                   = var.product
-  username               = local.db_master_usr
-  create_random_password = true
-  random_password_length = 12
-  port                   = 5432
+  name     = var.rds_instance_name
```

Comment:
Do we need to pass in this variable from Jira module? With some conditional logic around if snapshot_identifier is present.

Reply:
Actually there is no extra logic needed. RDS instance name is optional value for RDS. Identifier is the mandatory one.  Default values works for both Snapshot and without the snapshot restore.

Reply:
The current solution will break for non-snapshot deployment, i.e. no db will be created.

Reply:
@yzha645 , thanks for bringing this up... you are right, we need to consider this. This is fixed now.